scripts/generate_road_colors.py

```diff
@@ -0,0 +1,211 @@
+from colormath.color_conversions import convert_color
+from colormath.color_objects import LabColor, LCHabColor, SpectralColor, sRGBColor, \
+XYZColor, LCHuvColor, IPTColor
+from colormath.color_diff import delta_e_cie2000
+import numpy
```

Comment:
Or rather not, as I am unsure which (if any) gems are installed with ruby. I opened #1815 to cover this.